simulation/config/example.yaml

```diff
@@ -34,7 +34,7 @@ observation:
     step: 4
 
 simulation:
-  output: output/sim_example.fits
+  output: sim_example.fits
```

Comment:
Now that you changed this line, you also need to change this line in CI:

https://github.com/lusee-night/luseepy/blob/114426535dd43ce7370392cd22394121ede2eb40/.github/workflows/luseepy-test.yml#L26

So that it reads the file from the right place.